windows-installer/VERSION

```diff
@@ -0,0 +1 @@
+v1-13
```

Comment:
Should we be reading this file from `generate_sdl_agent_exe.ps1`?

Reply:
We read from the release.bat then pass the version to `generate_sdl_agent_exe .ps1`. That way i have the version recorded in `export_to_sponge_config ` so that Rapid can have it later in the Custom Fields. Though i can let `generate_sdl_agent_exe.ps1` read directly from the VERSION but after i saw ops-agent also let the build script retrieve the version then pass it.. I think we shall do it consistenly.

Reply:
The version for building Linux packages is read directly by the build scripts in this repo (it's also read by our build infrastructure, but that can be done in this case as well). So I'd suggest keeping `release.bat` the way you have it now, but also read this from `generate_sdl_agent_exe.ps1` to make the build more self-contained. You can still allow overriding the version from outside of the script, by using this file to simply set the default value of the flag.

Reply:
Done.
![image](https://user-images.githubusercontent.com/8354694/98043535-5f6a6a00-1df3-11eb-8e6e-f31acfdae1d7.png)